package.json

```diff
@@ -16,10 +16,12 @@
   },
   "scripts": {
     "test": "mocha build/test",
+    "test-individual": "mocha build/test -g 'loading function'",
```

Comment:
Did you mean to hardcode the `'loading function'` here?

Reply:
FYI this is pretty easy to do today with `npm run test -- -g 'my test'` so I am not sure it is worth adding a script for this

Reply:
Sounds good. I wanted to document how to do this somewhere.

I removed the script but added a note on how to do this in the contributing section.